deprivation_index_by_zipcode.r

```diff
@@ -0,0 +1,25 @@
+library(tidyverse)
+
+# read deprivation index
+library(tidyverse)
+
+# Deprivation index
+dep_index <- readRDS(gzcon(url('https://github.com/cole-brokamp/dep_index/raw/master/ACS_deprivation_index_by_census_tracts.rds'))) 
+
+
+# ZCTA data 
+ZCTA_data<- read.table("https://www2.census.gov/geo/docs/maps-data/data/rel/zcta_tract_rel_10.txt", header = TRUE,sep = ',',
+                       colClasses = c('ZCTA5' = 'character' , 'GEOID' = 'character'))

```

Comment:
We need to only select `ZCTA5` and `GEOID` here so that the other variables do not get written into the data file (for example, `POPPT`, `HUPT`, etc)